lib/context_adapter_helper.js

```diff
@@ -23,7 +23,9 @@
 
 'use strict';
 
+var caLogger = require('logops');
 var caConfig = require('./context_adapter_configuration');
+caLogger.setLevel(caConfig.LOGOPS_LEVEL);
```

Comment:
In fact, the way you suggest is the way it is done in the STH component :) And the same caLogger instance (once initialised) is passed to the rest of the modules of the app... This has the advantages of configuring the the logging just in one place and the disadvantage of making the JS files more dependant than it is really needed...

I guess you vote for the first approach, right? I mean, having the logging configured in just one place, right? :)